db/migrate/20150625164705_make_merged_at_datetime.rb

```diff
@@ -0,0 +1,6 @@
+class MakeMergedAtDatetime < ActiveRecord::Migration
+  def change
+    remove_column :hubstats_pull_requests, :merged_at
+    add_column :hubstats_pull_requests, :merged_at, :datetime
```

Comment:
Okay, I didn't see closed_at before, which is is a working example of this scenario.  

created_at and updated_at are handled specially by Rails so I wasn't looking at those as an example of this.

:+1: